packages/lesswrong/components/ultraFeed/UltraFeedCommentItem.tsx

```diff
@@ -121,12 +121,16 @@ const styles = defineStyles("UltraFeedCommentItem", (theme: ThemeType) => ({
   },
   verticalLine: {
     width: 0,
-    borderLeft: `4px solid ${theme.palette.grey[300]}`,
+    borderLeft: `4px solid ${theme.palette.grey[300]}ac`,
```

Comment:
I can't tell if that's a typo or a real change

Reply:
ah, that's a typo. adjusting the transparency is only for the green

Reply:
either uncommitted or unfixed